### PR TITLE
v0.22

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Review by Emilgardis — APPROVED** [review]
bors r+

**Comment by Emilgardis** [comment]
bors cancel

File: src/generate/register.rs [hunk 1]
```diff
@@ -357,17 +357,14 @@ pub fn fields(
...
                     }
                 } else {
-                    (0, None)
-                };
-                let suffixes: Vec<_> = match index {
-                    Some(ix) => ix,
-                    None => (0..de.dim).map(|i| (first + i).to_string()).collect(),
+                    0
                 };
+                let suffixes: Vec<_> = de.indexes().collect();
```
Comment (Emilgardis) [inline src/generate/register.rs:367]: are these changes intended to be included?

Comment (burrbull) [reply src/generate/register.rs:367]: Yes. It's just code cleanup.
